Cython/Includes/libcpp/string.pxd

```diff
@@ -58,19 +58,23 @@ cdef extern from "<string>" namespace "std" nogil:
         size_t length()
         void resize(size_t)
         void resize(size_t, char c)
+        void shrink_to_fit()
         size_t capacity()
         void reserve(size_t)
         void clear()
         bint empty()
-        iterator erase(iterator position)
-        iterator erase(const_iterator position)
-        iterator erase(iterator first, iterator last)
-        iterator erase(const_iterator first, const_iterator last)
+        iterator erase(iterator, iterator)
+        iterator erase(iterator)
+        iterator erase(const_iterator, const_iterator)
+        iterator erase(const_iterator)
+        string& erase(size_t, size_t) except +
+        string& erase(size_t) except +
+        string& erase() except +
 
         char& at(size_t) except +
         char& operator[](size_t)
-        char& front()  # C++11
-        char& back()   # C++11
+        char& front()
+        char& back()
```

Comment:
This documentation suggests that the comments are correct:
https://www.cplusplus.com/reference/string/string/

Reply:
They are correct, but to_string is also C++11 and wasn't explicitly marked as such. There may even be more unmarked C++11 features. I'm not sure what your stance on backwards compatibility is, but in 2021, C++11 should be the minimum-supported version in most compilers unless you specify otherwise via compiler flags.

Reply:
Ok, let's keep them out then to avoid the confusing mix of versioned and unversioned entries.